index.html

```diff
@@ -130,11 +130,15 @@
           <a href='https://operationcode.org'><img src="images/medal_navy.png" alt="opcodelogo"></a>
         </div>
       </section>
-    <!-- <div class="footer">
+
+<div id="footer">
+    <footer>
+        <span>Built by <a href="http://deployopcode.io/" class="underline">Deploy</a> The Dev Shop for <a href="https://operationcode.org/" class="underline">Operation Code.</a></span>
+    </footer>
     <footer>
-      <span>Built by <a href="http://deployopcode.io/">Deploy</a> The Dev Shop for <a href="https://operationcode.org/">Operation Code</a>
+        <a class="underline" href="https://github.com/OperationCode/vettec">Pull Request are welcome.</a>
```

Comment:
<img width="1278" alt="screen shot 2017-04-13 at 12 34 58 pm" src="https://cloud.githubusercontent.com/assets/20747057/25021215/44787bdc-2046-11e7-9121-058835f6efe5.png">

sorry accidentally removed the last comment , here is the latest changes